src/main/java/ru/yandex/qatools/embed/postgresql/PostgresProcess.java

```diff
@@ -189,6 +189,12 @@ protected void onBeforeProcess(IRuntimeConfig runtimeConfig)
             throws IOException {
         super.onBeforeProcess(runtimeConfig);
         PostgresConfig config = getConfig();
+
+        final File     dbDir   = config.storage().dbDir();
+        if (dbDir.exists() && dbDir.listFiles() != null && dbDir.listFiles().length > 0) {
```

Comment:
This is legal, because Initially de.flapdoodle.embed.process.runtime.AbstractProcess#onBeforeProcess() is completely empty